lib/utils/movie_repository.dart

```diff
@@ -0,0 +1,53 @@
+import 'dart:convert';
+import 'package:flutter/services.dart';
+import 'package:movie_app_jsalinas/models/genres.dart';
+import 'package:movie_app_jsalinas/models/movie.dart';
+
+class MovieRepository {
```

Comment:
this class looks like it could be separated into an Interface and the respective implementation of the Interface, in 2 different files

Reply:
solved